ERCS/erc-7750.md

```diff
@@ -0,0 +1,80 @@
+---
+eip: 7750
+title: Decentralized Employment System
+description: A blockchain-based employment system that records employment history.
```

Comment:
```suggestion
description: An employment system that records employment history.
```

Almost all ERCs are blockchain-based, so mentioning it here takes up valuable space in your description. You should use more of the available space here to describe your proposal.